### ISSUE TITLE
Consider disabling GBC

### ISSUE THREAD

**Issue by versus666** [ZeroK-RTS/Zero-K#2137]
Unit_global_build_command.lua is quite a problem and I wonder why no one complained about that.

TL;DR : this dipshit think it knows better than me what's good for me and do truckload bullshit before crashing. And I'm still polite.

So, this widget replace _Central Build AI_ made by Xponen. At first it seems great, many nice options and all, very promising. But start any game and the horror begin. It's even more visible on chicken games.

First, it tries to do the auto orders everywhere on the map. And auto orders are repairing and resurrecting. And where are the targets of such acts ? Frontlines/battlefields/etc, in other words hot zones where it's expected to die. Sure, it's nice to have your combat units auto healed, it's really efficient when steamrolling the enemy bases and I was surprised when I saw it do that so well. But when I saw, again and again, groups of workers running toward the enemy bases just to get slaughtered by their defenses I became annoyed. 'Auto-Chicken' option ? Triggers randomly. Do what ? I really don't know. And after the 5 times in the same game I lost 15+ worked doing stupid things like that just because I didn't babysit each one of them to tell it to NOT go kissing the enemy units I was quite pissed. And it's like that every game. Multiply that by half a dozen games each day since it's been introduced to ZK and you can measure how I can be angry. No more 'Auto-Reclaim/Resurrect' & 'Auto-Repair' options used, too bad.
Not fun.

Then, direct orders. It knows better than following them. It plays it's own agenda and die miserably. Because having 4 half-built missile towers is surely better than 2 completely built and 2 remaining jobs to do. Reclaiming ressources I'm missing like I just told it ? No, it's better to keep building with ressources I don't have even if there is plenty on the ground nearby ! Babysitting again. Babysitting stubborn gormless plonker robots.
Not fun.

The only way to USUALLY keep it from doing that is to set area commands. Fine, why not ? But, because it have to recompute god knows why FOR EACH WORKER INVOLVED, it freeze the game badly ! With the less CPU intensive setting, 10 workers : nearly half a second lag, 20 workers : 2 seconds lags, 60 Workers : make a coffee, 100+ workers : go to the coffee shop, ask the nice waitress for her number, date her and Spring *may* unfreeze before your first child will be born.
Even placing a build job on a location where there is wreck(s) exhibit such behavior...
Not fun.
(at least the Spring part)

Alt-s to erase jobs in an area. Works, just sometimes it's stuck in removal selection mode and you can't build until suddenly, by luck or by dint of clicking everywhere while shouting your lungs out, it stops.
Not fun.

Sometimes, it crash. In fact anything that change the queue may make it crash. Manual commands or from other widgets like unit_dynamic_avoidance. After the 5th crash in the first 10 minutes of the game you become really efficient at using Alt-F11. And you disable _unit_dynamic_avoidance_ but now your workers die stupidly and you're still frustrated. Not the best trade-off.
Not fun.

All of this with a Linux Mint 17.1 64 bit updated, with an i5 760 @ 2.80GHz, 12 gigs of RAM, Spring 103.0 and the most updated ZK.

Log will be attached, need some sleep.

**Comment by GoogleFrog** [comment]
Sounds like the widget should be disabled, not merely hidden behind menu option.
